components/match2/wikis/leagueoflegends/match_summary.lua

```diff
@@ -146,14 +137,22 @@ end
 function CustomMatchSummary._createBody(match)
 	local body = MatchSummary.Body()
 
-	if match.dateIsExact or (match.date ~= _EPOCH_TIME_EXTENDED and match.date ~= _EPOCH_TIME) then
+	if match.dateIsExact or match.timestamp ~= DateExt.epochZero then
 		-- dateIsExact means we have both date and time. Show countdown
 		-- if match is not epoch=0, we have a date, so display the date
 		body:addRow(MatchSummary.Row():addElement(
 			DisplayHelper.MatchCountdownBlock(match)
 		))
 	end
 
+	if tonumber(match.liquipediatier) == 1 and match.timestamp > BIG_MATCH_START_TIME then
```

Comment:
is there a reason it only gets enabled starting may?
what if it would be added for older torunaments?

Reply:
It may be changed in the future, but this is it for now